sormas-app/app/src/main/java/de/symeda/sormas/app/pathogentest/edit/PathogenTestEditFragment.java

```diff
@@ -0,0 +1,123 @@
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.app.pathogentest.edit;
+
+import android.util.Log;
+import android.view.View;
+
+import java.util.List;
+
+import de.symeda.sormas.api.Disease;
+import de.symeda.sormas.api.facility.FacilityDto;
+import de.symeda.sormas.api.sample.PathogenTestResultType;
+import de.symeda.sormas.api.sample.PathogenTestType;
+import de.symeda.sormas.app.BaseEditFragment;
+import de.symeda.sormas.app.R;
+import de.symeda.sormas.app.backend.caze.Case;
+import de.symeda.sormas.app.backend.common.DatabaseHelper;
+import de.symeda.sormas.app.backend.facility.Facility;
+import de.symeda.sormas.app.backend.sample.PathogenTest;
+import de.symeda.sormas.app.caze.edit.CaseNewActivity;
+import de.symeda.sormas.app.caze.read.CaseReadActivity;
+import de.symeda.sormas.app.component.Item;
+import de.symeda.sormas.app.component.controls.ControlPropertyField;
+import de.symeda.sormas.app.component.controls.ValueChangeListener;
+import de.symeda.sormas.app.databinding.FragmentPathogenTestEditLayoutBinding;
+import de.symeda.sormas.app.util.DataUtils;
+
+import static android.view.View.GONE;
+
+public class PathogenTestEditFragment extends BaseEditFragment<FragmentPathogenTestEditLayoutBinding, PathogenTest, PathogenTest> {
+
+    private PathogenTest record;
+    private Case sourceCase;
+
+    // Enum lists
+
+    private List<Facility> labList;
+    private List<Item> testTypeList;
+    private List<Item> diseaseList;
+    private List<Item> testResultList;
+
+    // Instance methods
+
+    public static PathogenTestEditFragment newInstance(PathogenTest activityRootData) {
+        return newInstance(PathogenTestEditFragment.class, null, activityRootData);
+    }
+
+    private void setUpControlListeners(FragmentPathogenTestEditLayoutBinding contentBinding) {
+
+    }
+
+    private void setUpFieldVisibilities(FragmentPathogenTestEditLayoutBinding contentBinding) {
+
+    }
+
+    // Overrides
+
+    @Override
+    protected String getSubHeadingTitle() {
+        return getResources().getString(R.string.heading_pathogen_test_summary);
+    }
+
+    @Override
+    public PathogenTest getPrimaryData() {
+        return record;
+    }
+
+    @Override
+    protected void prepareFragmentData() {
+        record = getActivityRootData();
+        testTypeList = DataUtils.getEnumItems(PathogenTestType.class, true);
+        diseaseList = DataUtils.getEnumItems(Disease.class, true);
+        testResultList = DataUtils.getEnumItems(PathogenTestResultType.class, true);
+        labList = DatabaseHelper.getFacilityDao().getLaboratories(true);
+    }
+
+    @Override
+    public void onLayoutBinding(FragmentPathogenTestEditLayoutBinding contentBinding) {
+        setUpControlListeners(contentBinding);
+
+        contentBinding.setData(record);
+//        contentBinding.setCaze(sourceCase);
```

Comment:
Please remove commented code that is no longer used, especially if it's a leftover from copying a class